### PR TITLE
Fix Lists Page Images Accessibility

### LINKED ISSUE #4633: Testing Lists Page Images for Accessibility
# Page to Test:

Home page - https://openlibrary.org/lists

# Steps

Steps for this task are outlined in #4521 

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #4633 

Combined the `cover` Div and `name` Div because the cover image for a list is a book cover of the book present in that list but the alt attribute says alt="Cover of: listname" and according to @bpmcneilly the alt text needs to inform users where they will go if they activates the link. So the best solution was to bring the cover inside the name and make the `alt=""`.

**For more Reference:** https://github.com/internetarchive/openlibrary/issues/4633#issuecomment-783681715

### Screenshot

image
### Stakeholders

@bpmcneilly

### CODE REVIEW

File: openlibrary/templates/lists/home.html [hunk 1]
```diff
@@ -30,15 +30,16 @@ <h2>$_('Active Lists')</h2>
     <!-- FOR EACH LIST -->
     $for list in lists:
         <div class="listEntry">
-            <div class="cover">
```
Comment (jdlrobson) [inline openlibrary/templates/lists/home.html:33]: Could you summarize in the pull request description why we needed to change the HTML markup here to address an accessibility issue?

Comment (Yashs911) [reply openlibrary/templates/lists/home.html:33]: Updated

Comment (jdlrobson) [reply openlibrary/templates/lists/home.html:33]: thank you

**Comment by jdlrobson** [comment]
@mekarpeles can you take a look at this one? I can't seem to get http://localhost:8080/lists to render without an error... :/

**Comment by bpmcneilly** [comment]
@Yashs911 - this looks perfect from an accessibility perspective on my end! Unless there are any other issues others flag, I think this is good to merge.

**Comment by RayBB** [comment]
Posting a screenshot of what the focus indicator looks like just for reference during discussion:
image
